books/free-programming-books-fr.md

```diff
@@ -122,7 +122,8 @@
 
 ### Chaîne de blocs / Blockchain
 
-* [Maîtriser Bitcoin: Programmer la chaîne de blocs publique](https://maitriser-ca.github.io/LivreMaitriserBitcoin/) - Andreas M. Antonopoulos, Serafim Dos Santos (asciidoc, html)
+* [Maîtriser Bitcoin: Programmer la chaîne de blocs publique](https://github.com/maitriser-ca/LivreMaitriserBitcoin) - Andreas M. Antonopoulos, Serafim Dos Santos (asciidoc, html)
```

Comment:
not http://bitcoin.maitriser.ca/ ?

Reply:
Hum... I'll have to re-evaluate this PR and link within the rendered pages (aka *.maitriser.ca) a clear way to these repo URLs instead.

I'll rePR soon.

Thanks for the enlightenment!